### ISSUE TITLE
Azure Devops Services Rest API - Git-Commits-Overview missing

### ISSUE THREAD

**Issue by fifthgearonline** [MicrosoftDocs/feedback#3643]
The overview page for Git Commits is missing in the Azure DevOps Services REST API documentation.

https://docs.microsoft.com/en-us/rest/api/azure/devops/git/Commits.yml?view=azure-devops-rest-6.0

![image](https://user-images.githubusercontent.com/21365801/122093878-2191e300-cdd1-11eb-9d35-a852e5dd5414.png)

**Comment by rootsmusic** [comment]
@fifthgearonline Your link is not found.

**Comment by fifthgearonline** [comment]
> @fifthgearonline Your link is not found.

Yeah, that's the entire reason I reported the issue. The page doesnt exist, when it should.

**Comment by rootsmusic** [comment]
@fifthgearonline I can load https://docs.microsoft.com/en-us/rest/api/azure/devops/git/commits?view=azure-devops-rest-6.0
